content/md/assets/full-automatic-pam-migration.md

```diff
@@ -0,0 +1,112 @@
+1. Check and test the assets migration
+1. Set transformations (*optional*, only if you use real transformations in the PAM)
+
+
+## _Step 1_ | Install & Setup the migration tool
+We built a dedicated tool [CSVToAsset](https://github.com/akeneo/CsvToAsset) to migrate your PAM assets to new Asset Manager assets.
+
+To install and setup this tool, refer to its [readme file](https://github.com/akeneo/CsvToAsset/blob/master/README.md).
+
+## _Step 2_ | Launch the PAM assets migration
+Then, you will launch one single command of the [CSVToAsset](https://github.com/akeneo/CsvToAsset) tool and your PAM assets will be migrated in the 4.0 new Asset Manager.
+
+Run `./bin/migrate.php <family-code> <path-to-ee-installation>` with the family-code you choose for the asset family created.
+
+This command will:
+1. Export the PAM assets in CSV into a temporary folder
+1. Create dedicated API credentials
+1. Migrate the assets data:
+    - Create a new asset family with the code family-code
+    - Export the PAM assets with all their data (properties, reference and variation files) in CSV files
+    - Merge the exported Akeneo PAM CSV files in one single file
+    - Import the merged file into Akeneo PIM in the new asset format
+1. Migrate the former Akeneo PAM product attributes into new Asset product attributes and linked them to the new asset family with the code family-code
+
+## _Step 3_ | Check and test the assets migration
+Yeah, well done, your assets are successfully migrated to the new Asset Manager!
+
+But we strongly recommend you to make some checks and tests to verify that everything is ok and discover more our new Asset Manager.
+
+### To view all your assets
+1. Go to the `Assets` menu
+1. Your asset family will be selected by default and all your assets will automatically appear in the grid
+1. You can search for an asset or display an asset by clicking on it
+
+:::info
+If you want to go further, please read this article [Create and display assets](create-and-display-assets.html).
+:::
+
+### To view how is structured your asset family
+1. Go the `Assets` menu`
+1. Click on the button `Edit family` at the top right of the grid
+1. View the family attributes in `Attributes` tab, you can translate the attributes labels for each locale
+1. View the family properties in `Properties` tab, you can translate the family labels for each locale
+
+:::info
+If you want to go further, please read this article [Manage asset families](manage-asset-families.html)
+:::
+
+### To view the assets linked to a product
+1. Click on a product on the product grid
+1. Click on the `Assets` tab, all the assets linked to this product will be displayed
+
+:::info
+If you want to go further, please read this article [Work with asset in an asset collection atribute](work-with-assets-in-an-asset-collection-attribute-ee-only.html).
+:::
+
+## _Step 4_ | Set transformations (*optional*)
+In the new Asset Manager, **transformations are no more mandatory**.
```

Comment:
you can also add a sentence to say "if you add dummy transformations before and don't need the variation attributes, you can remove them at this stage". Warning: be clear about the fact that they may lose some data if they had manually uploaded some variations

Reply:
oui, effectivement, je vais l'ajouter